snapcraft/extensions/kde_neon_6.py

```diff
@@ -174,7 +174,10 @@ def get_root_snippet(self) -> Dict[str, Any]:
                     "command-chain": ["snap/command-chain/hooks-configure-desktop"],
                 }
             },
-            "layout": {"/usr/share/X11": {"symlink": "$SNAP/kf6/usr/share/X11"}},
+            "layout": {
```

Comment:
I expect this to fix errors we engine resources and locales, which could also be fixed by
export QTWEBENGINE_RESOURCES_PATH="$SNAP_DESKTOP_RUNTIME/usr/share/qt6/resources"
export QTWEBENGINE_LOCALES_PATH="$SNAP_DESKTOP_RUNTIME/usr/share/qt6/translations/qtwebengine_locales"

I am not aware of a need to add additional layout mappings, but there are other qt6 specific directories.  Should we remap the lib ones just in case?
$ find /snap/kf6-core22/11/ -name "qt6"
/snap/kf6-core22/11/etc/qt6
/snap/kf6-core22/11/usr/include/x86_64-linux-gnu/qt6
/snap/kf6-core22/11/usr/lib/qt6
/snap/kf6-core22/11/usr/lib/x86_64-linux-gnu/qt6
/snap/kf6-core22/11/usr/lib/x86_64-linux-gnu/qt6/examples/corelib/threads/queuedcustomtype/lib/x86_64-linux-gnu/qt6
/snap/kf6-core22/11/usr/lib/x86_64-linux-gnu/qt6/examples/dbus/remotecontrolledcar/lib/x86_64-linux-gnu/qt6
/snap/kf6-core22/11/usr/share/qt6

Reply:
The library directories are taken care of through the `LD_LIBRARY_PATH` additions in `get_part_snippet` below. If something gets added to the qt6 or kf6 content snaps for `/etc/qt6` we might need to manage it, but for now this looks right to me.